department-of-veterans-affairs/content-build
# Vye folder build

## Description
Created file structure in vets-website for new application "verify-your-enrollment"

Pushing updated registry.json with new reference to verify your enrollment app

related vets-website PR - https://github.com/department-of-veterans-affairs/vets-website/pull/26058


closes __   
relates to __  

## Testing done & Screenshots
No testing other than making sure new app shows up in local environment. App does show.


## QA steps

What needs to be checked to prove this works?  
- User can navigate to /education/verify-your-enrollment/
- note- this is currently blank as it is a place holder.
What needs to be checked to prove it didn't break any related things?  
n/a, nothing deleted only added new reference to new app
What variations of circumstances (users, actions, values) need to be checked?  n/a

1. Do this
2. Then
3. Then validate Acceptance Criteria from issue


## Acceptance criteria


## Definition of done
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)/ Github PR for vetswebsite has been linked
